crates/sui-core/src/consensus_handler.rs

```diff
@@ -133,6 +132,19 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
         let mut transactions = vec![];
         let timestamp = consensus_output.sub_dag.commit_timestamp;
 
+        let epoch_start = self
+            .epoch_store
+            .epoch_start_config()
+            .epoch_start_timestamp_ms();
+        let timestamp = if timestamp < epoch_start {
```

Comment:
Just want to make sure, I assume epoch_start > all consensus commit timestamps from previous epochs?

Reply:
Yes :)